### PR TITLE
Clarified the "End user update experience"

### PR DESCRIPTION
**Pull request by joeyglocke**
clarified which updates cause end users to manually accept an update to the app

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-your-lob-apps.md [hunk 1]
```diff
@@ -122,6 +122,20 @@ To update an app, developers should continue to follow [step 1](#step-1-develop-
...
 
+### End user update experience
+
+In most cases, after you complete an app update the new version will automatically appear for end users. However, there are some updates to the <a href="https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema" target="_blank">Microsoft Teams manifest</a> that require user acceptance to complete:
+
+* a bot was added or removed
+* an existing bot's "botId" property changed
+* an existing bot's "isNotificationOnly" property changed
+* the bot's "supportsFiles" property changed
+* a Messaging Extension was added or removed
```
Comment (JohanFreelancer9) [inline Teams/manage-your-lob-apps.md:133]:
```suggestion
* a Messaging extension was added or removed
```

File: Teams/manage-your-lob-apps.md [hunk 2]
```diff
@@ -122,6 +122,20 @@ To update an app, developers should continue to follow [step 1](#step-1-develop-
 
 You can update the app through the tenant app catalog. To do this, in the Microsoft Teams admin center, go to **Teams apps** > **Manage apps**. In the list of apps, click the app name, and then click **Update**. Doing this replaces the existing app in the tenant app catalog, and all app permission policies and app setup policies remain enforced for the updated app.
 
+### End user update experience
+
+In most cases, after you complete an app update the new version will automatically appear for end users. However, there are some updates to the <a href="https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema" target="_blank">Microsoft Teams manifest</a> that require user acceptance to complete:
```
Comment (JohanFreelancer9) [inline Teams/manage-your-lob-apps.md:127]:
```suggestion
In most cases, after you complete an app update the new version will automatically appear for end users. However, there are some updates to the <a href="https://docs.microsoft.com/microsoftteams/platform/resources/schema/manifest-schema" target="_blank">Microsoft Teams manifest</a> that require user acceptance to complete:
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**